sys/sys/mbuf.h

```diff
@@ -331,7 +343,7 @@ struct mbuf {
 	int32_t		 m_len;		/* amount of data in this mbuf */
 	uint32_t	 m_type:8,	/* type of data in this mbuf */
 			 m_flags:24;	/* flags; see below */
-#if !defined(__LP64__)
+#if __SIZEOF_SIZE_T__ < 8
```

Comment:
I think this one wants to be POINTER == 4 (or ILP32) given the caddr_t above? (There's an odd number of pointers and int32's up to this point)

Reply:
And there's a uint64_t's worth of padding that we could explicitly include for CHERI128 instead.